utils/createSiteMeta.ts

```diff
@@ -0,0 +1,60 @@
+const type = 'website'
+const url = 'https://nft.kodadot.xyz'
+const title = 'KodaDot - Kusama NFT Market Explorer'
+const description = 'Creating Carbonless NFTs on Kusama'
+const image = 'https://nft.kodadot.xyz/kodadot_card_root.png'
```

Comment:
the fullpath are default values?

Reply:
yes but you actually made a good point, i think i will rewrite this function as a plugin -> then i will additionally have access to context & $config and can write something like `const url = this.$config.baseUrl` as default values 😸